apps/laboratory/app/library/bitcoin-wallet-button/page.tsx

```diff
@@ -31,9 +32,9 @@ ThemeStore.setModal(modal)
 
 export default function Wagmi() {
   return (
-    <>
+    <InitializeBoundary>
```

Comment:
Why no wrapping the the pages in `layout.tsx` instead?

Reply:
Tried but cannot guarantee appkit is initialized since it's created on each page :|